tests/fixtures/markdown-with-closed-html-tags.md

```diff
@@ -0,0 +1,4 @@
+# SDS API Documentation
+<hr><br>This documentation describes SDS APIs, that can be broadly categorized into : Index, Ops, ResolveBy. For any questions please contact acp-cs-sds-engineering@adobe.com<br><br>
```

Comment:
I think you meant for this file to contain `<hr/> <br/> and <img/>` tags. Right now it's a duplicate of `tests/fixtures/markdown-with-open-html-tags.md`

Reply:
Yep. Good catch.